pypy/doc/contributing.rst

```diff
@@ -77,14 +77,15 @@ Some ideas for first contributions are:
 * Missing language features - these are listed in our `issue tracker`_
 
 .. _nightly builds: https://buildbot.pypy.org/nightly/
-.. _issue tracker: https://foss.heptapod.net/pypy/pypy/issues
+.. _issue tracker: https://github.com/pypy/pypy/issues/
 
 Source Control
 --------------
 
-PyPy's main repositories are hosted here: https://foss.heptapod.net/pypy.
+PyPy's main repositories are hosted here: https://github.com/pypy.
```

Comment:
```suggestion
PyPy's main git repositories are hosted here: https://github.com/pypy,
and legacy repositories are hosted here https://foss.heptapod.net/pypy.
```

Reply:
Thanks for reviewing this! I've done as suggested, and slightly changed the description in the following paragraph.